assets/constants.js

```diff
@@ -264,12 +264,20 @@ var constants = {
     "text": "中文(繁體)"
   }],
 
+  "//" : [
+    "The priority values are used to determine what language should be ",
```

Comment:
Oohh ... ranking the languages of the world? :P

I think this is fraught, though. Would Singlish have a higher or lower priority than Hinglish? Would French or Italian have a higher priority? Hard to decide.

We might want a more algorithmic way of doing this, e.g. put student's language and exploration language (and perhaps "related languages", like Hinglish to Hindi) near the top, and then follow this with other languages in random order.